docs/vs-code-extension-guide/master.adoc

```diff
@@ -11,22 +11,19 @@ include::topics/templates/document-attributes.adoc[]
 = Visual Studio Code Extension Guide
```

Comment:
I thought you were going to change this to sentence case.

Reply:
Was planning to leave alone for and later do all documents. No need to wait, though.

Reply:
I thought you said that sentence case was already being used in the attributes for the titles.